power-platform/admin/cleanup-asyncoperationbase-table.md

```diff
@@ -73,6 +73,9 @@ For synchronous workflows, we recommend enabling **Keep logs for workflow jobs t
 
 :::image type="content" source="media/bulk-deletion-keep-logs.png" alt-text="Set Workflow Log Retention to keep logs for workflow jobs that encountered errors.":::
 
+> [!NOTE]
+> The bulk deletion job may be automatically canceled if the environment is in [Administration mode](/power-platform/admin/admin-mode#set-administration-mode).
```

Comment:
Would the right fix be to allow bulk delete jobs to execute in admin only mode? Is there a specific reason why we want to cancel bulk delete jobs when an org is in admin-only mode?

Reply:
Sorry for the delayed response. I encountered this when troubleshooting an environment that was in administration mode and didn't realize it was causing the issue. I added the note in case anyone runs into the same issue in the future. Allowing bulk delete jobs in admin only mode might be a better option.